app/src/main/res/layout/activity_bmi_calculator.xml

```diff
@@ -10,16 +10,16 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:gravity="center_horizontal"
-        android:text="BMI Calculator"
+        android:text="@string/bmi_calculator"
         android:textSize="32sp"/>
 
     <ToggleButton
         android:id="@+id/activity_main_toggleunits"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:checked="true"
-        android:textOff="Imperial Units"
-        android:textOn="Metric Units" />
+        android:textOff="@string/imperial_units"
+        android:textOn="@string/metric_units" />
```

Comment:
Fixed the error, and added the missing strings.
Source:
https://www.youtube.com/watch?v=vXjJyLpydxg
https://www.youtube.com/watch?v=YaWmNhlOXYk

Reply:
Good.